action-allowedlist/approved.json

```diff
@@ -2,12 +2,14 @@
     {
         "actionLink": "actions/checkout",
         "actionVersion": "5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f",
-        "tag": "v2.3.4"
+        "tag": "v2.3.4",
+        "deprecated": true
```

Comment:
@andonyns With your code, there was a problem that the deprecated action was not also approved, therefore the job failed. I was able to move past that by having the deprecated action in both files. Then I thought: why have two files? Just update this one to mark something as deprecated.